dev-games/tiled/tiled-1.11.0.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 Gentoo Authors
```

Comment:
Why did you modify the copyright years? Usually you wouldnt change what the previous ebuild did so this would be 1999-2024.

Reply:
I thought here should be the year the particular ebuild version made

Reply:
https://mgorny.pl/articles/new-gentoo-copyright-policy-explained.html#copyright-ownership

> With the simplified method, the copyright line will usually look like:
> 
> \# Copyright START[-END] Gentoo Authors
> 
> With the complete method, it would be:
> 
> \# Copyright START[-END] LARGEST-OWNER [and others]
> 
> START indicates the earliest year that the listed owners claim copyright to the file. END indicates the latest.